packages/suite/src/hooks/suite/useRedirect.ts

```diff
@@ -0,0 +1,80 @@
+import { Account } from '@wallet-types';
+import { BuyTradeQuoteRequest } from 'invity-api';
+import { useActions } from '@suite/hooks/suite';
+import * as routerActions from '@suite-actions/routerActions';
+import * as coinmarketBuyActions from '@wallet-actions/coinmarketBuyActions';
+
+export const useRedirect = () => {
```

Comment:
rename to `useCoinmarketRedirect`

Reply:
I renamed it but left it where it is - it is used by out of wallet coinmerket-redirect view.